epochStart/metachain/systemSCs.go

```diff
@@ -1245,4 +1284,6 @@ func (s *systemSCProcessor) EpochConfirmed(epoch uint32) {
 		"epoch", epoch,
 		"maxNodes", s.maxNodes,
 	)
+
+	s.flagCorrectLastUnjailed.Toggle(epoch == s.correctLastUnJailEpoch)
```

Comment:
In staking.go this flag is toggled by condition >=, here is ==

Reply:
yes. as the reset should happen only once. this is correct.